Week-1/day2/favMusician/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Arijit Singh Fan Webpage</title>
```

Comment:
title element content is identical to the h1 tag.
The title element could be more descriptive of the page content, such as "Arijit Singh: Biography and Music".